images/chromium-headful/Dockerfile

```diff
@@ -32,7 +32,9 @@ RUN set -eux; \
     make -j$(nproc); \
     make install;
 
-FROM ghcr.io/m1k1o/neko/chromium:3.0.6 AS neko
+#FROM ghcr.io/m1k1o/neko/chromium:3.0.6 AS neko
+FROM ghcr.io/raiden-staging/neko/chromium:3.0.6-kernel-editv1 AS neko
+# ^--------- edited + rebuilt neko:chromium to disable host only on clipboard events
```

Comment:
Hey @raiden-staging I've forked neko over here: https://github.com/onkernel/neko

Could you open a PR on that repo with this change?